website/conferences/views.py

```diff
@@ -215,6 +215,14 @@ def conference_view(**kwargs):
         )
         projects = Node.find(query)
         submissions = projects.count()
+        downloads_list = [
```

Comment:
This is a rather expensive operation. Have you tried this locally with a a significant number of nodes for each conference to see if the load times are acceptable?